test/src/test/java/hudson/model/JobTest.java

```diff
@@ -286,4 +291,35 @@ private static void tryConfigDotXml(JenkinsRule.WebClient wc, int status, String
         p.renameTo("different-name");
     }
 
+    @Issue("JENKINS-30502")
+    @Test public void testRenameTrimsSpace() throws Exception {
```

Comment:
Please put annotation on new line, coding style that you see is inventing by one guy and has bad readability :)